.github/workflows/linters

```diff
@@ -0,0 +1,59 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```

Comment:
- [ ] As suggested by the previous reviewer, It would be better to add the file extension (.yml) to this linter configuration file and I hope this will fix the issue :+1: :+1: 
    - Kindly Rename this file from ```linters``` to ```linters.yml``` as it is on [linter config files](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) :pray: 

**From**
![image](https://user-images.githubusercontent.com/102757126/205450504-871890bc-c21c-44ee-a02c-816e2fabcc56.png)
**to**
![image](https://user-images.githubusercontent.com/102757126/205450477-40267034-3647-422c-8f6b-560586b02dee.png)